09-redux/classworks/Malkov_Aleksei/src/resources/reducer.js

```diff
@@ -0,0 +1,34 @@
+const initialState = {
```

Comment:
let's use `combineReducers` instead of 1 global reducer

Reply:
https://github.com/paralect/startupsummer2020/pull/121/commits/3dda899bf4f017b97199d1d35270a9e8e5daa43e
split to 2 reducers and combineReducer